### PR TITLE
contrib: Implement side-by-side comparison tool

### PR DESCRIPTION
**Pull request by ahsanbarkati**
---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/7974)

### CODE REVIEW

**Review by manishrjain — APPROVED** [review]
:lgtm: Nice work on this script. I have some suggestions. Call it contrib/sxs/sxs.go.

> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/7974#-:-MgGgLI70bhI5YTA59ZS:b-j2hq2p)* status: all files reviewed, 3 unresolved discussions (waiting on @ahsanbarkati)

___
*[contrib/verification-tool/vtool.go, line 1 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/7974#-MgGgEoH5WebZM_prcO5:-MgGgEoH5WebZM_prcO6:b-prne0e) ([raw file](https://github.com/dgraph-io/dgraph/blob/bac8e98ec0d6d94027be0e40bb98ffc890016c2d/contrib/verification-tool/vtool.go#L1)):*
> ```Go
> package main
> ```

The tool can be called sxs.go -- Side by Side comparisons are very common.
___
*[contrib/verification-tool/vtool.go, line 40 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/7974#-MgGgC5_C12Kl8A5dPAr:-MgGgA1l5ZQ-a9JcCfNa:bqnn2zf) ([raw file](https://github.com/dgraph-io/dgraph/blob/bac8e98ec0d6d94027be0e40bb98ffc890016c2d/contrib/verification-tool/vtool.go#L40)):*
> ```Go
> func main() {
> 	filePath := flag.String("log-file", "", "Path of the log file")
> 	alphaSrc := flag.String("alpha-src", "", "GRPC endpoint of alpha")
> ```

alpha-left
___
*[contrib/verification-tool/vtool.go, line 41 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/7974#-MgGgD0N8UrbNh2CwHD2:-MgGgD0N8UrbNh2CwHD3:b-px3r5k) ([raw file](https://github.com/dgraph-io/dgraph/blob/bac8e98ec0d6d94027be0e40bb98ffc890016c2d/contrib/verification-tool/vtool.go#L41)):*
> ```Go
> 	filePath := flag.String("log-file", "", "Path of the log file")
> 	alphaSrc := flag.String("alpha-src", "", "GRPC endpoint of alpha")
> 	alphaTest := flag.String("alpha-test", "", "GRPC endpoint of master alpha")
> ```

alpha-right

**Review by manishrjain — COMMENTED** [review]
or sbs/sbs.go -- side by side

> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/7974#-:-MgGgRsB0mkADMJHSZhM:b60w0lo)* status: all files reviewed, 3 unresolved discussions (waiting on @ahsanbarkati)

File: contrib/sbs/sbs.go [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+	alphaRight string
+	countOnly  bool
+	queryFile  string
+	numGo      int
+}
+
+func init() {
+	Sbs.Cmd = &cobra.Command{
+		Use:   "sbs",
+		Short: "A tool to do side-by-side comparision of dgraph clusters",
```
Comment (NamanJain8) [inline contrib/sbs/sbs.go:64]: Typo in `comparison`

File: contrib/sbs/sbs.go [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+	flags := Sbs.Cmd.Flags()
+	flags.StringVar(&opts.logPath,
+		"log-file", "", "Path of the alpha log file to replay")
+	flags.StringVar(&opts.alphaLeft,
+		"alpha-left", "", "GRPC endpoint of left alpha")
+	flags.StringVar(&opts.alphaRight,
+		"alpha-right", "", "GRPC endpoint of right alpha")
+	flags.BoolVar(&opts.countOnly,
+		"counts-only", false, "Only get the count of all predicates in the left alpha")
```
Comment (NamanJain8) [inline contrib/sbs/sbs.go:76]: Lets do that on both the alphas?

**Review by NamanJain8 — APPROVED** [review]

**Pull request merged**